packages/roosterjs-editor-types/lib/interface/ListStyleMetadata.ts

```diff
@@ -0,0 +1,10 @@
+import { BulletListType } from '../enum/BulletListType';
+import { NumberingListType } from '../enum/NumberingListType';
+
+/**
+ * Represents the metadata of the style of a list element
+ */
+export default interface ListStyleMetadata {
```

Comment:
This type is only used by VLIst and VListItem, and we don't expect other developer can access it. So you can declare it in VLIstItem, and make an internal export, no need to export from here.